### PR TITLE
Eventmode

### PR DESCRIPTION
**Pull request by optimumtact**
Adds an eventmode define

Fast acting planetary atmos on all turfs, with breathablea tmosphere

deletes all cleanables after 30 ticks

power to all areas, default gas mixes on everything

free vending machines everywhere

economy doesn't tick

breathing and diseases in life disabled to prevent some life stuff

No nightshift either tbh 

disease/advanced disease datums delet eon creation

rot components delete on creation to prevent miasma

thoughts on other improvemetns we can maek welcome

Assumes low memory mode is also enabled (no z levels other than centcom and one map level)

cutting down trees disabled

no animal breeding

no smugglers satchels

no random round events

### CODE REVIEW

**Comment by actioninja** [comment]
Disable miasma processing

**Comment by optimumtact** [comment]
miasma is a gas isn't it? the changes to atmos will clear it within a few seconds

**Comment by optimumtact** [comment]
rot components disabled

**Comment by optimumtact** [comment]
ignore the travis failures btw, it's unit tests getting sad cause im deleting stuff

File: _maps/_basemap.dm [hunk 1]
```diff
@@ -1,4 +1,4 @@
-//#define LOWMEMORYMODE //uncomment this to load centcom and runtime station and thats it.
+#define LOWMEMORYMODE //uncomment this to load centcom and runtime station and thats it.
```
Comment (PKPenguin321) [inline _maps/_basemap.dm]: 👀

**Comment by wesoda25** [comment]
Interesting

**Comment by nero1024** [comment]
What is it that makes eventmode even useful? All I see that it does is simplify atmos, lighting, etc. to lower CPU usage.

**Comment by optimumtact** [comment]
you answered your own question.

It's a test mergeable branch for allowing large population events to run more smoothly

**Comment by Detective-Google** [comment]
would it be possible/ in scope to instead have this as a button IG for admemes? i.e. 'toggle eventmode for next round'?

**Comment by PKPenguin321** [comment]
I don't think that's what it's for

**Comment by wesoda25** [comment]
Rip

**Pull request closed without merging**